### PR TITLE
Optimize preci & ngihtly tests

### PR DESCRIPTION
**Pull request by mengfei25**
1. Additional PRs apply script for windows
2. Checkout the corresponding torch & torch-xpu-ops versions for wheel tests
3. Don't need source DL-Essential for nightly wheel tests

### CODE REVIEW

File: .github/workflows/_linux_ut.yml [hunk 1]
```diff
@@ -65,19 +65,21 @@ jobs:
...
-          git status && git show -s
-          git submodule sync && git submodule update --init --recursive
-          if [[ ${{ inputs.keep_torch_xpu_ops }} == 'true' ]]; then
-            echo "Don't replace torch-xpu-ops!"
-          else
-            rm -rf third_party/torch-xpu-ops && cp -r ../torch-xpu-ops third_party/
-            # Workaround for torch-xpu-ops ci test
-            sed -i "s/checkout --quiet \${TORCH_XPU_OPS_COMMIT}/log -n 1/g" caffe2/CMakeLists.txt
+          git clone https://github.com/pytorch/pytorch pytorch
+          if [ "${{ inputs.pytorch }}" != "nightly_wheel" ]; then
```
Comment (chuanqi129) [inline .github/workflows/_linux_ut.yml:70]: Does it mean we only apply those patch for ci and source build nightly test, is it right?

Comment (mengfei25) [reply .github/workflows/_linux_ut.yml:70]: yes

**Review by chuanqi129 — APPROVED** [review]
please launch a demand test to verify those change

**Review by chuanqi129 — APPROVED** [review]

**Comment by mengfei25** [comment]
Passed with nightly wheel https://github.com/intel/torch-xpu-ops/actions/runs/12292086940

**Comment by mengfei25** [comment]
The UT failures are not related with this change

**Pull request merged**